packages/terra-button/package.json

```diff
@@ -29,7 +29,7 @@
     "lint:scss": "stylelint src/**/*.scss",
     "test": "npm run test:jest && npm run test:wdio",
     "test:jest": "jest --config ../../jestconfig.js",
-    "test:wdio": "../../node_modules/.bin/wdio ../../node_modules/terra-dev-site/config/wdio/wdio.conf.js"
+    "test:wdio": "cross-env ../../node_modules/.bin/wdio ../../node_modules/terra-dev-site/config/wdio/wdio.conf.js"
```

Comment:
why was cross-env added?

Reply:
I am working through a window's machine and the cross-env enables me to run wdio script. But, After the conversation is resolved I will be deleting this file.